.github/workflows/build-release.yml

```diff
@@ -1,17 +1,22 @@
-name: Build Release
+name: build-release
 
-on: [ workflow_dispatch ]
+on:
+  release:
+    types:
+      - published
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK 8
-        uses: actions/setup-java@v1
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Set up JDK 16
+        uses: actions/setup-java@v2
```

Comment:
I'll need to check, though from what I could tell releases were already being generated with J16, unless Github branches were being dumb. I'll test in a bit

Reply:
Yeah, I'm pretty sure they do work, we just need to be absolutely sure because this method of publishing pushes releases directly to mod sites without local testing. I did compile 1.1.3 with Java 16 I think though.

Reply:
I'm 90% sure it's fine as long as you don't use any Java9+ featues

Reply:
I have tested it and it works. I could see no issues in the log or in game.